akka-actor/src/main/scala/akka/event/DeadLetterListener.scala

```diff
@@ -33,9 +33,16 @@ class DeadLetterListener extends Actor {
       val done = maxCount != Int.MaxValue && count >= maxCount
       val doneMsg = if (done) ", no more dead letters will be logged" else ""
       eventStream.publish(Info(rcp.path.toString, rcp.getClass,
-        s"Message [${message.getClass.getName}] $origin to $rcp was not delivered. [$count] dead letters encountered$doneMsg. " +
-          "This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' " +
-          "and 'akka.log-dead-letters-during-shutdown'."))
+        s"""
+           |Message [${message.getClass.getName}] from [$origin] to [$rcp] was not delivered.
+           |[$count] dead letters encountered $doneMsg.
+           |
+           |If this is not an expected behavior, then the [$rcp] may died unexpectedly,
+           |You can inspect the cause by adding supervision strategy to it's parent.
```

Comment:
I'm not sure this advice will help many. Default supervision for exceptions is to restart the actor and then there will be no deadLetters, and if the actor is stopped the parent supervision will not really see that unless it's watching.